Dockerfile

```
@@ -21,7 +20,6 @@ RUN	CGO_ENABLED=0 GOOS=linux go build -mod vendor -ldflags \
     "-w -s -X sigs.k8s.io/aws-encryption-provider/pkg/version.Version=$TAG" \
     -o bin/aws-encryption-provider cmd/server/main.go
 
-FROM busybox AS aws-encryption-provider
-COPY --from=build /etc/ssl/certs/ /etc/ssl/certs/
+FROM ${BASE_IMAGE}
 COPY --from=build /go/src/sigs.k8s.io/aws-encryption-provider/bin/aws-encryption-provider /aws-encryption-provider
-ENTRYPOINT ["/aws-encryption-provider"]
+ENTRYPOINT ["/aws-encryption-provider"]
```

Since you're importing go-runner, why not use it as your entrypoint? See https://github.com/kubernetes/release/tree/master/images/build/go-runner
```suggestion
ENTRYPOINT ["/go-runner"]
CMD ["/aws-encryption-provider"]
```